cloudx/handler.go

```diff
@@ -232,6 +232,27 @@ func (h *SnakeCharmer) EnsureContext() (*AuthContext, error) {
 
 	if len(c.SessionToken) > 0 {
 		_, _ = fmt.Fprintf(h.verboseErrWriter, "You are authenticated as: %s\n", c.IdentityTraits.Email)
```

Comment:
Potentially move this to after the check? Would make more sense form a UX perspective.